sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -1013,10 +1013,10 @@ case class ShowCreateTableCommand(table: TableIdentifier) extends RunnableComman
     builder ++= s"CREATE$tableTypeString ${table.quotedString}"
 
     if (metadata.tableType == VIEW) {
-      if (metadata.schema.nonEmpty) {
-        builder ++= metadata.schema.map(_.name).mkString("(", ", ", ")")
-      }
-      builder ++= metadata.viewText.mkString(" AS\n", "", "\n")
+      showViewDataColumns(metadata, builder)
+      showTableComment(metadata, builder)
```

Comment:
how about renaming `showTableComment` to `showTableOrViewComment `?

Reply:
how about make it more simple, `showComment`?

Reply:
Looks ok.